src/Microsoft.AspNet.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -186,24 +187,29 @@ public GeneratedTagHelperContext()
         public string TagHelperContentTypeName { get; set; }
 
         /// <summary>
-        /// The name of the method used to write <see cref="ExecutionContextTypeName"/>.
+        /// The name of the property containing the <c>HtmlEncoder</c>.
         /// </summary>
-        public string WriteTagHelperAsyncMethodName { get; set; }
+        public string HtmlEncoderPropertyName { get; set; }
 
         /// <summary>
-        /// The name of the method used to write <see cref="ExecutionContextTypeName"/> to a specified
-        /// <see cref="System.IO.TextWriter"/>.
+        /// The name of the method used to convert a <c>TagHelperContent</c> into a <see cref="string"/>.
         /// </summary>
-        public string WriteTagHelperToAsyncMethodName { get; set; }
+        public string TagHelperContentGetContentMethodName { get; set; }
 
         /// <summary>
-        /// The name of the property containing the <c>HtmlEncoder</c>.
+        /// The property accessor to indicate if the tag helper's content has been modified.
         /// </summary>
-        public string HtmlEncoderPropertyName { get; set; }
+        public string TagHelperOutputIsContentModifiedPropertyName { get; set; }
 
         /// <summary>
-        /// The name of the method used to convert a <c>TagHelperContent</c> into a <see cref="string"/>.
+        /// The property accessor for the tag helper's output content.
         /// </summary>
-        public string TagHelperContentGetContentMethodName { get; set; }
+        public string TagHelperOutputContentPropertyName { get; set; }
+
+        /// <summary>
+        /// The name of the method on the property <see cref="ExecutionContextOutputPropertyName"/> used to retrieve
+        /// tag helper child content.
```

Comment:
I just added the docs to be semi-nice to users. We actually don't have any plans to doc core Razor stuff like this :smile: